src/main/scala/sbtghactions/WorkflowJob.scala

```diff
@@ -28,7 +28,7 @@ final case class WorkflowJob(
     env: Map[String, String] = Map(),
     oses: List[String] = List("ubuntu-latest"),
     scalas: List[String] = List("2.13.6"),
-    javas: List[JavaSpec] = List(JavaSpec.temurin("11")),
+    javas: List[JavaSpec] = List(JavaSpec.temurin("11"), JavaSpec.temurin("17")),
```

Comment:
So I don't think this is actually relevant. `WorkflowJob` is not manually used by users, instead its populated by this plugin but I changed it to reflect the change to `githubWorkflowJavaVersions`.